source/features/hide-user-forks.tsx

```diff
@@ -7,16 +7,34 @@ function addSourceTypeToLink(link: HTMLAnchorElement): void {
 	link.search = String(search);
 }
 
+const skipUrlsWithType = ':not([href*="&type="])';
+
+const selectors = [
+	// User repos
+	`a[href$="?tab=repositories"]:is([href^="/"], [href^="${location.origin}/"])${skipUrlsWithType}`,
```

Comment:
Currently `skipUrlsWithType` will never match. To make it work for e.g. search strings (`q=`) probably this

```suggestion
	`a[href*="?tab=repositories"]:is([href^="/"], [href^="${location.origin}/"])${skipUrlsWithType}`,
```